app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt

```diff
@@ -0,0 +1,64 @@
+package com.marcossalto.targetmvd.network.models
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import com.squareup.moshi.ToJson
+
+@JsonClass(generateAdapter = true)
+data class User (
+    @Json(name = "id") val id: Int,
+    @Json(name = "email") val email: String,
+    @Json(name = "uid") val uid: String,
+    @Json(name = "provider") val provider: String,
+    @Json(name = "avatar") val avatar: Avatar,
+    @Json(name = "first_name") var firstName: String? = "",
+    @Json(name = "last_name") var lastName: String? = "",
+    @Json(name = "username") val username: String,
+    @Json(name = "gender") var gender: String,
+    @Json(name = "push_token") val pushToken: String?,
+    @Json( name ="allow_password_change") val allowPasswordChange: Boolean? = false
+)
+
+@JsonClass(generateAdapter = true)
+data class Avatar(
+    @Json(name = "url") val url: String? = null,
+    @Json(name = "normal") val normal: Normal = Normal(null),
+    @Json(name = "small_thumb") val smallThumb: SmallThumb = SmallThumb(null)
+    )
```

Comment:
this parenthesis should be at the start of the line

Reply:
Thanks for your suggestion